docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -151,3 +139,56 @@ fed-minion          <none>              Ready
 **The cluster should be running! Launch a test pod.**
 
 You should have a functional cluster, check out [101](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/walkthrough/README.md)!
+
+**Run time addition/deletion of minions (or nodes).**
+
+* Create following minion.json file on kubernetes master node:
+
+```json
+{
+  "id": "fed-node1",
+  "kind": "Minion",
+  "apiVersion": "v1beta1",
+  "resources": {
+    "capacity": {
+      "cpu": 1,
+      "memory": 1073741824
+    },
+  },
+  "labels": {
+    "name": "fed-node1-label",
+  },
+}
+```
+
+Now create a minion object internally in your kubernetes cluster by running:
+
+```
+$ kubectl create -f minion.json
+
+$ kubectl get nodes
+NAME                LABELS              STATUS
+fed-node1           name=fed-node1-label     Unknown
```

Comment:
I think --machines= should be removed from the guide entirely.  It makes clusters harder to grow, and it's impossible to shrink.  Lets just evict it entirely from the guide. Only show dynamic clusters...

and yes, we are supposed to call them nodes, lets fix that too...